apps/src/lib/kits/maker/boards/microBit/MicroBitBoard.js

```diff
@@ -58,32 +64,20 @@ export default class MicroBitBoard extends EventEmitter {
    */
   openSerialPort() {
     const portName = this.port ? this.port.comName : undefined;
-    const SerialPortType = serialPortType(false);
-
-    /** @const {number} serial port transfer rate */
-    const SERIAL_BAUD = 57600;
+    const SerialPortType = serialPortType();
```

Comment:
`serialPortType` was originally built to distinguish the SerialPort (Maker App) from the ChromeSerialPort (Chrome App). Since WebSerial is replacing ChromeApp and `openSerialPort` will only be used for the MakerApp, we should be able to just write this as
```
const serialPort = new SerialPort(portName, {baud . . .})
```
and delete line 67.

Reply:
Thanks! Also also removed reference to `serialPortType()` about where `portType` is assigned.
I have a question about `SerialPort`... in `browserChecks.js`, I see that `SerialPort` is global and 'Maybe provided by the Code.org Browser'.
I added the `global SerialPort` in this file at the top. Lmk if this is appropriate - thanks!